scripts/irods/test/test_iadmin.py

```diff
@@ -1612,7 +1640,7 @@ def test_non_admins_are_not_allowed_to_invoke_iadmin_lg__issue_6188(self):
         self.user0.assert_icommand(['iadmin', 'lg'], 'STDERR', ['-830000 CAT_INSUFFICIENT_PRIVILEGE_LEVEL'])
 
     def test_non_admins_are_not_allowed_to_invoke_iadmin_set_delay_server__issue_5249(self):
-        self.user0.assert_icommand(['iadmin', 'set_delay_server', lib.get_hostname()], 'STDERR', ['-13000 SYS_NO_API_PRIV'])
+        self.user0.assert_icommand(['iadmin', 'set_delay_server', lib.get_hostname()], 'STDERR', ['-830000 CAT_INSUFFICIENT_PRIVILEGE_LEVEL'])
```

Comment:
Was this test just out of date?

Reply:
The error code returned is actually changing here because `iadmin` is checking the user's privilege level and preventing `rodsuser`s from using it. This was necessary to maintain the "security" afforded by SimpleQuery since everything was changed to GenQuery.